src/components/SourceFooter.css

```diff
@@ -12,50 +12,39 @@
   user-select: none;
 }
 
-html:not([dir="rtl"]) .source-footer .command-bar {
-  float: right;
+.source-footer__command-bar {
```

Comment:
talked with @Garbee offline and the simple solution is to rename the class `command-bar` to something else.